wallet-new/src/Cardano/Wallet/API/V1/Errors.hs

```diff
@@ -91,35 +92,51 @@ type ErrorExample = Value
 sample :: [WalletError]
 sample =
   [ NotEnoughMoney 1400
-  , OutputIsRedeem "b10b24203f1f0cadffcfd16277125cf7f3ad598983bef9123be80d93"
-  , SomeOtherError "foo" 14
-  , MigrationFailed "migration"
+  , OutputIsRedeem "b10b242...be80d93"
```

Comment:
Don't you think we should use the long representation? As this is intended to be a "real-world" representation of an error, it would be a shame if somehow somebody could misinterpret that we are rendering addresses in such short notation. What do you think?

Reply:
This one gave me a hard time thinking actually. The reason why I truncated here is because these samples are used for documentation and, since we render the examples from those samples, the particular one yield to a "wide" example completely messing up the whole formatting of the table. Could be fixed with a bit of css, yet having simply a more human-readable address here seemed more _legit_ to me :/

Reply:
hehe ok, fair enough -- definitely not a big deal, but I'm glad it gave you food for thoughts 😉